lib/migration.pm

```diff
@@ -104,19 +104,7 @@ sub deregister_dropped_modules {
     my @all_addons = grep($_, split(/,/, get_var('SCC_ADDONS', '')));
     for my $name (grep($_, split(/,/, $droplist))) {
         record_info "deregister $name", "deregister $name module and remove it from SCC_ADDONS";
-        if ($name eq 'ltss') {
-            if (check_var('SLE_PRODUCT', 'hpc')) {
-                remove_suseconnect_product('SLE_HPC-LTSS');
-            } elsif (is_sle('15+') && check_var('SLE_PRODUCT', 'sles')) {
-                remove_suseconnect_product('SLES-LTSS');
-            } else {
-                zypper_call 'rm -t product SLES-LTSS';
-                zypper_call 'rm sles-ltss-release-POOL';
-            }
-        }
-        else {
-            remove_suseconnect_product(get_addon_fullname($name));
-        }
+        remove_suseconnect_product(get_addon_fullname($name));
```

Comment:
I think we need think about sles11, this will cause sles11 case failed since 11 doesn't support SUSEConnect.

Reply:
yes, agree with you, updated the pr, thanks.